include/oneapi/dpl/pstl/utils_ranges.h

```diff
@@ -56,6 +56,17 @@ using __value_t = decltype(oneapi::dpl::__internal::get_value_type<_R>(0));
 template <typename _Proj, typename _R>
 using __key_t = ::std::remove_cv_t<::std::remove_reference_t<::std::invoke_result_t<_Proj&, __value_t<_R>>>>;
 
+template <class T, class = void>
+struct range_has_raw_ptr_iterator : ::std::false_type
```

Comment:
uglification

Reply:
Could you please clarify how you would like this line to be formatted? Should I move the inheritance to a new line?

Reply:
@adamfidel, uglification means adding undescores in front of all the names, like this:
```
template <class _T, class = void>
struct __range_has_raw_ptr_iterator : ::std::false_type
```

Note also that we typically use `typename` instead of `class` for template parameters.

Reply:
Thanks for the clarification @akukanov. I updated the names to have double underscores and fixed the class/typename issue.

@MikeDvorskiy would you mind re-approving?